config.example.json

```diff
@@ -5,5 +5,6 @@
   "accessSecret": "TWITTER ACCESS SECRET",
   "slackWebhook": "SLACK INCOMING WEBHOOK",
   "slackChannel": "OPTIONAL SLACK CHANNEL TO SEND TO",
-  "keywords": "keywords,your want,to track"
+  "keywords": "keywords,your want,to track",
+  "keywordsIgnored": "keywords,you do not want,to track"
```

Comment:
json supports lists of strings, why don't we use them rather than splitting this string every time?